src/main/java/org/folio/circulation/support/http/client/PageLimit.java

```diff
@@ -7,6 +7,7 @@
   private static final PageLimit NO_PAGE_LIMIT = new PageLimit(null);
   private static final PageLimit ONE = limit(1);
   private static final PageLimit ONE_THOUSAND = limit(1000);
+  private static final PageLimit TEN_THOUSAND = limit(10000);
```

Comment:
What led to choosing 10 000 rather than 1 000?

Reply:
The field is removed.